Update active-directory-passwords-policy.md

softwarecraft opened the pull request:
Update link to Azure AD PowerShell module as prior link redirects to page that recommends using the new link provided

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

softwarecraft committed:
Update active-directory-passwords-policy.md

Update link to Azure AD PowerShell module as prior link redirects to page that recommends using the new link provided

softwarecraft committed:
Update active-directory-passwords-policy.md

Remove localization
